qml/ConnectionMenu.qml

```diff
@@ -184,10 +184,11 @@ Item {
                     Layout.columnSpan:  2
                     Layout.fillWidth: true
                     onEditingFinished: {
-                        if (input == "0.0.0.0" || input == "localhost") {
+                        if (text === "0.0.0.0" || text === "localhost") {
```

Comment:
should this be `udpIp.text` (as you have on line 191)?

Reply:
Or maybe 191 should be just `text`. Looking for consistency or understanding if there is some reason for the difference.